src/components/accession2/edit/ViabilityModal.tsx

```diff
@@ -15,23 +15,33 @@ export interface ViabilityDialogProps {
   onClose: () => void;
   reload: () => void;
   setNewViabilityTestOpened: React.Dispatch<React.SetStateAction<boolean>>;
+  changeTab: (event: React.SyntheticEvent, newValue: string) => void;
```

Comment:
since we aren't really using the event, maybe we should remove it here and in the Accession2View file? sorry for this last nit.